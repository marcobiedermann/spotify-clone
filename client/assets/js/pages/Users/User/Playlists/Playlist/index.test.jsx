import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import { PlaylistPage } from '.';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <MemoryRouter>
        <PlaylistPage />
      </MemoryRouter>,
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
