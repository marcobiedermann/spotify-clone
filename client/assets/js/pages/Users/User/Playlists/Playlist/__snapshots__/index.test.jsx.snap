// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <figure>
    <figcaption>
      <a
        href="/users//playlists/"
        onClick={[Function]}
      >
        
      </a>
    </figcaption>
  </figure>
  <p>
    Playlist
  </p>
  <h1 />
  <p>
    Created by
     
    <a
      href="/users/"
      onClick={[Function]}
    >
      
    </a>
     
    Â·
     
     
    songs
  </p>
  <p>
    <button
      className="button"
      type="button"
    >
      Play
    </button>
  </p>
  <table>
    <tbody />
  </table>
</div>
`;
